### PR TITLE
Update guidelines.md

### PR DESCRIPTION
**Pull request by ypriverol**
I have added a description to a couple of Rules.

### CODE REVIEW

**Comment by ypriverol** [comment]
@jgriss do not merge this yet.

File: guidelines.md [hunk 1]
```diff
@@ -84,8 +85,7 @@ Fields: all
 
 Name: The data repository is part of [ProteomeXchange](http://www.proteomexchange.org/)  
 Category: Bronze  
-Description:    
-Fields:  Proteomics, ProteoGenomics
```
Comment (jgriss) [inline guidelines.md]: Fields should remain

File: guidelines.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ Fields: Proteomics, ProteoGenomics
 
 Name: The mass spectra are provided in the HUPO-PSI standard file format mzML.  
 Category: Gold   
-Description:
+Description: mzML is a HUPO-PSI data format that store mass spectrometry data (e.g mass spectra, chromatograms) using controlled vocabulary terms. mzML, is a well tested open-source and XML-based format.
```
Comment (jgriss) [inline guidelines.md]: We could even add the link to the mzML spec / website here. Markdown syntax is fully supported by the viewer for description and examples.

File: guidelines.md [hunk 3]
```diff
@@ -103,10 +103,10 @@ Description: This relates to the search results. As most proteomics search engin
 by the search engine should be uploaded to the same repository as the mass spectrometer's raw data.
 Fields: Proteomics, ProteoGenomics  
 
-Name: The Quantitative information of is made available in the mzTab file format.  
+Name: The Quantitative information of is made available in the HUPO-PSI standard file format mzTab.   
 Category: Gold  
-Description:   
-Fields: Proteomics, ProteoGenomics  
+Description: mzTab is a HUPO-PSI standard file fomat that store proteomics and metabolomics identification and quantification results including: Peptide spectrum matches, Peptides, metabolites and proteins.
```
Comment (jgriss) [inline guidelines.md]: Should we add a link to the mzTab page?

File: guidelines.md [hunk 4]
```diff
@@ -128,7 +128,7 @@ Fields: ProteoGenomics
 
 Name: Data should be deposited in one of the following archives: Metabolights, MassIVE  
 Category: Silver   
-Description:     
+Description: Metabolights and MassIVE are the two major archives for metabolomics data in the world. These two archives implements standard protocols for data submission and dissemination of metabolomics data.
```
Comment (jgriss) [inline guidelines.md]: Add links?

File: guidelines.md [hunk 5]
```diff
@@ -243,12 +242,12 @@ Fields: all
...
 Name: The protein sequence database used including the exact version/date of download (if a release version is not available) are described  
 Category: Bronze    
-Description:   
+Description: During the protein identification step, the spectrum identification search engine algorithm commonly uses a  protein sequence FASTA database from public resources (e.g. ENSEMBL, Uniprot). If the original FASTA file from the resource is not transform/change and the release number of the file is not available; the exact version/date of the downloaded FASTA file should be provided.  
 Fields: Proteomics, ProteoGenomics  
 
 Name: The method used to create a decoy database is specified  
 Category: Bronze   
-Description:   
+Description: During the peptide identication step, commonly the search engine use a decoy database to statistically access the queality of the peptide identifications. Multiple methods are available to generate the decoy databases (e.g reverse, random, etc). We recommended to mention the stategy use to genrate the decoy database in the main manuscript.
```
Comment (jgriss) [inline guidelines.md]: Corrected some typos:

```suggestion
Description: During the peptide identification step, the search engine commonly uses a decoy database to statistically access the quality of the peptide identifications. Multiple methods are available to generate decoy databases (e.g reverse, random, etc). We recommended to mention the strategy used to generate the decoy database in the main manuscript.    
```

File: guidelines.md [hunk 6]
```diff
@@ -263,7 +262,7 @@ Fields: Proteomics, ProteoGenomics
 
 Name: The search engine(s) including their exact versions is/are specified  
 Category: Bronze  
-Description:  
+Description: During the identification step a search engine tool is used to map the obtained mass spectra to the corresponding peptide sequence. In this rule, we recommended to report in the main manuscript the software version.
```
Comment (jgriss) [inline guidelines.md]:
```suggestion
Description: During the identification step a `search engine` is used to map the obtained mass spectra to the corresponding peptide sequence. This recommendation highlights the importance of this step in the analysis process. Therefore, to obtain reproducible results it is imperative to mention the exact version of the used search engine.
```

File: guidelines.md [hunk 7]
```diff
@@ -369,7 +368,7 @@ Fields: all
 
 Name: Underlying data for all figures (where applicable) is made available  
 Category: Gold  
-Description:   
+Description: During the downstream analysis multiple figures commplements the final results of the manuscript. We recommend to make all figures available (where applicable) as supplementary information of the manuscript or as part of the dataset (e.g. Quality control reports).
```
Comment (jgriss) [inline guidelines.md]:
```suggestion
Description: During the downstream analysis multiple figures complement the final results of the manuscript. We recommend to make the underlying data of all figures available (where applicable) as supplementary information of the manuscript or as part of the dataset. Ideally, the code to create these figures should be made available as well (for example using [Jupyter notebooks](https://jupyter.org)).
```

**Review by jgriss — CHANGES_REQUESTED** [review]

**Review by ypriverol — COMMENTED** [review]
@jgriss all changes suggested has been included.

**Pull request merged**